src/implementations/twig/components/form-group/form-group.html.twig

```diff
@@ -27,6 +29,8 @@
 {% set _required = required|default(false) %}
 {% set _required_text = required_text|default('') %}
 {% set _optional_text = optional_text|default('') %}
+{% set _label_aria_required = label_aria_required|default('') %}
```

Comment:
I understand that we need to provide accessible markup and this is the way we can do it, but expecting that the users will always be able to set 4 internazionalided strings for setting required and optional text, plus an aria label for the two elements is really optimistic.